features/admin.roles.v2/pages/role-edit.tsx

```diff
@@ -52,6 +53,27 @@ const RoleEditPage: FunctionComponent<RoleEditPagePropsInterface> = (
 
     const [ roleId, setRoleId ] = useState<string>(undefined);
     const [ currentActiveTabIndex, setCurrentActiveTabIndex ] = useState<number>(0);
+    const userRolesV3FeatureEnabled: boolean = useSelector(
+        (state: AppState) => state?.config?.ui?.features?.userRolesV3?.enabled
+    );
```

Comment:
We can reuse `featureConfig` obtained in L52.

```suggestion

    const userRolesV3FeatureEnabled: boolean = featureConfig?.userRolesV3?.enabled

```